src/Apps/Order/Components/__tests__/AddressVerificationFlow.jest.tsx

```diff
@@ -96,16 +104,36 @@ describe("AddressVerificationFlow", () => {
       expect(screen.getByText("Edit Address")).toBeInTheDocument()
     })
 
-    it('calls onChosenAddress with the suggested address when "Use This Address" is clicked', async () => {
+    it("tracks that the validation modal was seen", async () => {
       renderWithRelay(
         {
           VerifyAddressType: () => mockResult,
         },
         undefined,
+        componentProps
+      )
+      await screen.findByText("Check your delivery address")
+
+      expect(trackEvent).toHaveBeenCalledTimes(1)
+      expect(trackEvent).toHaveBeenCalledWith({
+        action_type: "validationAddressViewed",
+        context_module: "ordersShipping",
+        context_page_owner_id: undefined,
```

Comment:
When we track checkout dead-ends, we set this field to the order ID. Should that also be the case here? (cc: @TMcMeans in case you've discussed this with @daytavares)